lib/widgets/rounded_button.dart

```diff
@@ -1,47 +1,48 @@
-import 'dart:developer';
 import 'package:flutter/material.dart';
-
 import 'package:fokus/utils/theme_config.dart';
 
 class RoundedButton extends StatelessWidget {
-  final IconData iconData;
+  final IconData icon;
   final String text;
   final Color color;
+	final bool dense;
+	final Function onPressed;
```

Comment:
Those are again the same three fields as in your button model, maybe use that + icon & dense?

Reply:
Honestly, I just cleaned some names and added missing button action. This widget is used only ones in @Evil00787's child panel page. Don't really know what to do with this. Here is usage snippet:
![obraz](https://user-images.githubusercontent.com/36575365/90314358-79429200-df13-11ea-8557-cd0ef0071f09.png)

Reply:
Oh, I know remember that at one point I used it. But with buttons you pretty much always a lot of customization. That's why there is dense parameter. Ops. 😞